src/PlusDataCollection/MicronTracking/vtkPlusMicronTracker.cxx

```diff
@@ -240,6 +246,25 @@ PlusStatus vtkPlusMicronTracker::InternalUpdate()
 #endif
   }
 
+  this->GetImage(this->FrameLeft, this->FrameRight);
+  vtkPlusDataSource* aSource(NULL);
+  for (int i = 0; i < this->GetNumberOfVideoSources(); ++i)
+  {
+    if (this->GetVideoSourceByIndex(i, aSource) != PLUS_SUCCESS)

```

Comment:
Please add a NotifyConfigure overload that checks that the user has created 2 and exactly 2 video sources for image output.